### DISCUSSION TITLE
posts/Jekyll-Chirpy-%ED%85%8C%EB%A7%88-%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%84%A4%EC%A0%95-%EA%B8%B0%EB%A1%9D-github-wiki-link/

### DISCUSSION THREAD

**Discussion by giscus** [zhyunk/zhyunk#4] [General]
# posts/Jekyll-Chirpy-%ED%85%8C%EB%A7%88-%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%84%A4%EC%A0%95-%EA%B8%B0%EB%A1%9D-github-wiki-link/

1. 프로필 이미지 & 게시글 이미지 주소 입력 형태 설정 _config.yml 1 2 # e.g. 'https://chirpy-img.netlify.app' img_cdn: "" img_cdn 값을 빈 값으로 두면 post 게시글에서 이미지 주소를 읽을 때 전체 url을 작성했다면 url을 그대로 읽고, /파일/경로를 적었을 때 깃허브 레포지토리를 루트로 해서 파일을 읽어온다. img_cdn 값을 입력해두면 post 게시글에서 http 등 프로토콜로 시작하는 경로를 입력할 경우 img_cdn값 다음에 프로토콜이 붙어서 해석되기 때문에 에러 발생

https://blog.zhyun.kim/posts/Jekyll-Chirpy-%ED%85%8C%EB%A7%88-%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%84%A4%EC%A0%95-%EA%B8%B0%EB%A1%9D-github-wiki-link/

**Comment by zhyunk** [comment]
### 코드 블럭에 liquid 코드 작성 할 경우
코드 전체를 <kbd>{% raw %}</kbd>와 <kbd>{% endraw %}</kbd>로 감싸줘야 한다.


> md 작성 예시
> ```liquid
> {% raw %}{% when 'tistory' %}
>     {%- capture url -%}
>         {{ site.social.links[1] }}
>     {%- endcapture -%}{% endraw %}
> ```
>
